getting-started.md

```diff
@@ -0,0 +1,142 @@
+## How to install and run Vega Wallet
+These instructions are written to be used in command line. Below, in the snippets, you'll see commands in blue text. Copy those instructions and paste them into your command line interface.
+
+### Download
+Download and save the zip file from Releases. We suggest you keep track of where you've saved the file, because that's where the command line interface will look for it. 
+https://github.com/vegaprotocol/go-wallet/releases
+
+**For MacOS:**
+
+Download `vegawallet-darwin-amd64.zip`
+
+When you open the file, you may need to change your system preferences for this specific instance, in order to run Vega Wallet. If you open the file from downloads, you may get a message saying “vegawallet-darwin-amd64” cannot be opened because it is from an unidentified developer.
+
+Click on the (?) help button, which will open a window that links you to the System Preferences, and instructs you how to allow this software to run.
+
+You’ll need to go to System Preferences > Security & Privacy > General, and choose “Open Anyway”.
+
+**For Windows:**
+
+Download `vegawallet-windows-amd64.zip`
+
+You may need to change your system preferences for this specific instance, in order to run Vega Wallet. If you open the file from downloads, you may get a message from Windows Defender saying “vegawallet-windows-amd64” cannot be opened because it is from an unidentified developer.
```

Comment:
```suggestion
You may need to change your system preferences for this specific instance, in order to run Vega Wallet. If you open the file from downloads, you may get a message from Windows Defender saying:

> “vegawallet-windows-amd64” cannot be opened because it is from an unidentified
> developer.
```